### PR TITLE
[DEV] adding `orthoani_fastas` param

### PR DESCRIPTION
**Pull request by Mxrcon**
Hey, as I mentioned on #3, I'm suggesting the creation of a param to take care of the fastas input on orthoani workflow. Here are my modifications suggesting that

I'm here to discuss these changes and make any modification requested and try new possibilities of implementing this parameter.

Hope we have a merge!
Kindly, Davi

### CODE REVIEW

**Review by abhi18av — APPROVED** [review]
Looks good to merge ✅ 

Thanks Davi! 😊

**Comment by Mxrcon** [comment]
Hey! I just found an typo on the yaml!

File: params/local.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ reads: "${baseDir}/data/full_data/Myc_Raw_Data/*_R{1,2}_001.fastq.gz"
 outdir: "${baseDir}/results"
 gbk_file: "${baseDir}/data/full_data/NZ_LT906469.1.fasta"
 orthoani_jar: "${baseDir}/resources/OAT_cmd.jar"
+orthoani_fastas = "${baseDir}/data/final_fasta/*fasta"
```
Comment (Mxrcon) [inline params/local.yml:5]: @abhi18av how do i proceed with the fix for this small typo?

Comment (abhi18av) [reply params/local.yml:5]: Open a PR 😉

Comment (Mxrcon) [reply params/local.yml:5]: ok!

**Pull request merged**